### PR TITLE
Added a new question to Git Quiz (Rename Branch)

### PR DESCRIPTION
**Pull request by Pratikxya**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -302,6 +302,16 @@ const gitQuiz = [
       "In the command line, you can use git checkout -b new-branch-name to create a new branch and switch to it",
     Link: "https://www.freecodecamp.org/news/git-and-github-for-beginners/",
   },
+  {
+    Question: "In Git, how do you rename current branch?",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:306]:
```suggestion
    Question: "In Git, how do you rename the current branch?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Pratikxya** [comment]
@jdwilkin4 , I have implemented the changes. 
Thanks.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**